src/qt/intro.cpp

```diff
@@ -23,7 +23,7 @@
 
 static const uint64_t GB_BYTES = 1000000000LL;
 /* Minimum free space (in GB) needed for data directory */
-static const uint64_t BLOCK_CHAIN_SIZE = 80;
+static const uint64_t BLOCK_CHAIN_SIZE = 102;
```

Comment:
This is on a node which started sync last week, so it shouldnt have many stale blocks.